2-download.html

```diff
@@ -1,7 +1,33 @@
 ---
-layout: redirect
-redirect: "/"
+layout: wiki
+title: "Download Jamulus"
 lang: "en"
 permalink: "/download"
 category: "hidden"
```

Comment:
Still hidden to not show it on google. However, as far as I remember, I've liked this page in the software.

Reply:
If this needs translation, it should be moved into /wiki/en (e.g. /wiki/en/misc) so the .po files can be created for each language. If for some reason it needs to exist in the root dir, it has to be moved anyway and a symlink created.

Reply:
Should there also be a link to replace the OS-specific ones on /wiki/en/misc/1-index.html?

Reply:
Not that up to date with the structural changes of the last few months. Probably symlinks should be created then?

Reply:
Not sure I follow... Basically, any docs requiring translation need to be in /wiki/en/ (or any sub-directory in there) as that's where po4a looks for translatable documents. Those that for whatever reason need to be located elsewhere are replaced with symlinks to the "original" files, which are moved into /wiki/en/. But if the location of a file doesn't need to be fixed, it's less messy to just move it into /wiki/en/. In any event, if symlinks are added, the po4a-add-language.sh script needs to be updated (I'd do that).

Reply:
Hmm. Since it’s since the file is linked in the software, it should be in the root. So we‘d need to link it.

Reply:
What do you mean by "linked in the software"?

Reply:
Sorry, I thought I linked upgrade.html to download.html because of https://github.com/jamulussoftware/jamulus/pull/1980

however, jamulus.io/download and jamulus.io/upgrade should link to a download page.

Reply:
With all the language versions of upgrade.html and download.html the root folder is going to get quite cluttered.

Reply:
>  jamulus.io/download and jamulus.io/upgrade should link to this download page once it’s finished

Not sure I follow, but is it that we can put them in en/wiki and have them translated as normal, but if we want "special" URLs for them  (ie jamulus.io/download as opposed to jamulus.io/wiki/download) we need to have redirects in the root, is that right? These can be symlinks to the language versions though, no?

Reply:
> Not sure I follow, but is it that we can put them in en/wiki and have them translated as normal, but if we want "special" URLs for them (ie jamulus.io/download as opposed to jamulus.io/wiki/download) we need to have redirects in the root, is that right?

Correct

>These can be symlinks to the language versions though, no?

Yes. The thing is that we end up with another 10 files (symlinks) in the root folder - increasing in number if more languages are added.

Reply:
I see. And I'm pretty sure GH Pages doesn't do server-side redirects either, so I guess there's no alternative.

Reply:
Do you think it's worth cluttering the root? Or do we find a better way to solve this problem?

Reply:
Can't the link in the app be changed for the next release?

Reply:
It could. But we'd also like to have jamulus.io/download too...